source/Tutorials/Advanced/Simulators/Installation-Windows.rst

```diff
@@ -0,0 +1,196 @@
+You can then either install ``webots_ros2`` from the official released package, or install it from the last up-to-date sources committed on `Github <https://github.com/cyberbotics/webots_ros2>`_.
+
+The following commands must be run inside the WSL environment.
+
+.. tabs::
+
+    .. group-tab:: Install ``webots_ros2`` distributed package
+
+        Run the following command in a terminal.
+
+        .. code-block:: console
+
+            sudo apt-get install ros-{DISTRO}-webots-ros2
+
+    .. group-tab:: Install ``webots_ros2`` from sources
+
+        Create a ROS 2 workspace with its ``src`` directory.
+
+        .. code-block:: console
+
+            mkdir -p ~/ros2_ws/src
+
+        Source the ROS 2 environment.
+
+        .. code-block:: console
+
+            source /opt/ros/{DISTRO}/setup.bash
+
+        Retrieve the sources from Github.
+        
+        .. code-block:: console
+
+            cd ~/ros2_ws
+            git clone --recurse-submodules https://github.com/cyberbotics/webots_ros2.git src/webots_ros2
+
+        Install the package dependencies.
+
+        .. code-block:: console
+
+            sudo apt install python3-pip python3-rosdep python3-colcon-common-extensions
+            sudo rosdep init && rosdep update
+            rosdep install --from-paths src --ignore-src --rosdistro {DISTRO}
+
+        Build the package using ``colcon``.
+
+        .. code-block:: console
+
+            colcon build
+
+        Source this workspace.
+
+        .. code-block:: console
+
+            source install/local_setup.bash
+
+
+
+4 Launch the ``webots_ros2_universal_robot`` example
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+WSL doesn't support hardware acceleration (yet). Therefore, Webots should be started on Windows, while the ROS part is running inside WSL.
+To do so, the following commands must be run inside the WSL environment.
+
+First source the ROS 2 environment, if not done already.
+
+.. code-block:: console
+
+        source /opt/ros/{DISTRO}/setup.bash
+
+Setting the ``WEBOTS_HOME`` environment variable allows you to start a specific Webots installation (e.g. ``C:\Program Files\Webots``).
+Use the mount point "/mnt" to refer to a path on native Windows.
+
+.. code-block:: console
+
+        export WEBOTS_HOME=/mnt/c/Program\ Files/Webots
+
+If installed from sources, source your ROS 2 workspace, if not done already.
+
+.. code-block:: console
+
+        cd ~/ros2_ws
+        source install/local_setup.bash
+
+Use the ROS 2 launch command to start demo packages (e.g. ``webots_ros2_universal_robot``).
+
+.. code-block:: console
+
+        ros2 launch webots_ros2_universal_robot multirobot_launch.py
+
+
+5 RViz troubleshooting
+^^^^^^^^^^^^^^^^^^^^^^
+
+With recent versions of WSL2, RViz should work out of the box.
+
+You can check if it works correctly by running any example that uses RViz, for example:
+
+.. code-block:: console
+
+        ros2 launch webots_ros2_tiago robot_launch.py rviz:=true slam:=true
```

Comment:
```suggestion
        sudo apt install ros-{DISTRO}-slam-toolbox
        ros2 launch webots_ros2_tiago robot_launch.py rviz:=true slam:=true
```

Reply:
This demo doesn't seem to work...

Reply:
This is due to an update in the `diff_drive_controller` package. You can by-pass the default wrong parameter by adding a line in `/opt/ros/humble/share/webots_ros2_tiago/resource/ros2_control.yml` under `use_stamped_vel: false`.

```
use_stamped_vel: false
base_frame_id: "base_link"
```